spec/lib/tag_manager_spec.rb

```diff
@@ -80,7 +80,7 @@
 
     it 'returns false for string with irrelevant characters' do
       Rails.configuration.x.web_domain = 'domain.test'
-      expect(TagManager.instance.local_url?('https://domainn.test/')).to eq false
+      expect(TagManager.instance.local_url?('https://domain.test/')).to eq false
```

Comment:
It's quite likely this is wrong.

such examples should really use `example.com` / `example.net` / ... instead of random potentially public domains.

If you're trying to ensure that the thing on line 82 isn't the same as the thing on line 83, then `domain.example.com` and `domain.example.net` would probably be good choices.

Reply:
Tests will never make a real HTTP request.